# Bioinformatics Genomics Data Workshop

=======

This workshop is for any researcher who has data they want to analyze , and no prior computational experience is required. This hands-on workshop teaches basic concepts, skills and tools for working more effectively with data.

The focus of this workshop will be on working with genomics data and data management and analysis for genomics research. We will cover metadata organization in spreadsheets, data organization, connecting to and using advanced computing, the command line for sequence quality control and bioinformatics workflows, and R for data analysis and visualization. We will not be teaching any particular bioinformatics tools, but the foundational skills that will allow you to conduct any analysis and analyze the output of a genomics pipeline.

Participants should bring their laptops and plan to participate actively. By the end of the workshop learners should be able to more effectively manage and analyze data and be able to apply the tools and approaches directly to their ongoing research.

### Workshop structure

One dataset will be used throughout the workshop. We will start by introducing the dataset and the steps we'll go through for analysis.

### Dataset

In this workshop we're using data from Blount et al 2012 paper from Dr. Richard Lenski's Long Term Evolution Experiment.  

- [Overview of the dataset](dataset/01-intro-to-dataset.html)

### Workshop overview
- [Overview](00-overview.html)
- [Introducing the dataset and questions](dataset/01-intro-to-dataset.html)
- [Genomic data in the NCBI SRA database](dataset/02-examining-sra-runtable.html)

**Metadata and Data Organization**

- [Metadata Lesson](data-tidiness/01-tidiness.html)
- Think about and understand the types of data and metadata a sequencing experiment will generate
- Gain a general understanding of data organization

**Module 3:** [Using cloud computing for genomics](https://jasonjwilliamsny.github.io/cloud-genomics/)  

- understand what cloud computing is and why it's useful
- log in to remote computing resources

**Introduction to the command line**

- [Overview](shell/index.html)
- [The file system](shell/01_the_filesystem.html)
- [Searching files](shell/02_searching_files.html)

**Module 5:** Data wrangling and processing

- [Project Setup](https://github.com/datacarpentry/organization-genomics/blob/gh-pages/lessons/01_intro_organization.md)
- [Quality control of NGS data](https://github.com/JasonJWilliamsNY/wrangling-genomics/blob/gh-pages/lessons/00-readQC.md)
- [Automating a workflow](https://github.com/JasonJWilliamsNY/wrangling-genomics/blob/gh-pages/lessons/01-automating_a_workflow.md)
- [Variant calling workflow](https://github.com/JasonJWilliamsNY/wrangling-genomics/blob/gh-pages/lessons/02-variant-calling-workflow.md) **Optional lesson**

**Module 5:** [R for data analysis and visualization](https://github.com/datacarpentry/R-genomics)

- importing data in to R and using it for analysis and visualization


### Requirements

This workshop is hands-on, so participants are encouraged to use
their own computers to insure the proper setup of tools for an efficient workfl\
ow.
*These lessons assume no prior knowledge of the skills or tools*, but working
through this lesson requires working copies of the software described.
To most effectively use these materials, please make sure to install everything
*before* working through this workshop.
